src/coreclr/vm/exceptionhandling.cpp

```diff
@@ -8128,7 +8130,11 @@ extern "C" CLR_BOOL QCALLTYPE CallFilterFunclet(QCall::ObjectHandleOnStack excep
         // it will retrieve the framepointer for accessing the locals in the parent
         // method.
         dwResult = CallEHFilterFunclet(OBJECTREFToObject(throwable),
+#ifdef TARGET_X86
+                                       GetRegdisplayFP(pvRegDisplay),
+#else
                                        GetFrameRestoreBase(pvRegDisplay->pCallerContext),
+#endif
```

Comment:
We already have a better definition for this:

```suggestion
#ifdef USE_CURRENT_CONTEXT_IN_FILTER
                                       GetFrameRestoreBase(pvRegDisplay->pCurrentContext),
#else
                                       GetFrameRestoreBase(pvRegDisplay->pCallerContext),
#endif
```